gnc/sub8_perception/src/sub8_pcl/cv_utils.cc

```diff
@@ -8,6 +8,11 @@ cv::Point contour_centroid(Contour& contour) {
   return center;
 }
 
+bool larger_contour(const Contour &c1, const Contour &c2){
+	if(cv::arcLength(c1, true) > cv::arcLength(c2, true)) return true;
```

Comment:
Imagine [this](https://upload.wikimedia.org/wikipedia/commons/d/d9/KochFlake.svg)